acceptance_tests/utilities/sftp_utility.py

```diff
@@ -23,11 +23,15 @@ def __enter__(self):
     def __exit__(self, *_):
         self.ssh_client.close()
 
-    def get_all_print_files_after_time(self, period_start_time):
+    def get_all_files_after_time(self, period_start_time, suffix=""):
         files = self._sftp_client.listdir_attr(Config.SFTP_DIR)
         period = period_start_time.strftime('%Y-%m-%d')
-        return list(filter(lambda f: f'P_IC_ICL1_{period}' in f.filename
-                                     and period_start_time <= datetime.fromtimestamp(f.st_mtime), files))
+
+        f = list(filter(lambda f: f'P_IC_ICL1_{period}' in f.filename
```

Comment:
It would be nice not to use `filter`. It's generally considered a little harmful now that list comprehensions are more widely accepted:

```suggestion
        return [f for f in files if f'P_IC_ICL1_{period}' in f.filename
                                 and f.filename.endswith(suffix)
                                 and period_start_time <= datetime.fromtimestamp(f.st_mtime)]
```